test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10986,6 +10989,33 @@ def forward(self, x):
         self.run_test(Module(False), x, rtol=1e-3, atol=1e-6)
         self.run_test(Module(True), x, rtol=1e-3, atol=1e-6)
 
+    @skipIfUnsupportedMinOpsetVersion(16)
+    def test_grid_sample(self):
+        n, c, h_in, w_in, h_out, w_out = 1, 1, 3, 2, 2, 4
+
+        class Module(torch.nn.Module):
```

Comment:
naming: Same as above

Reply:
Renamed in 71f8806a497fbb237089939d72ac8efb0417c226